packages/server/src/arpa_reporter/routes/users.js

```diff
@@ -70,7 +70,7 @@ router.post('/', requireAdminUser, async (req, res, next) => {
             const updatedUser = await createUser(user);
             res.json({ user: updatedUser });
 
-            void sendWelcomeEmail(updatedUser.email, `${req.headers.origin}/arpa_reporter`);
+            sendWelcomeEmail(updatedUser.email, `${req.headers.origin}/arpa_reporter`);
```

Comment:
I think this was actually here to address a lint error that we had set up in the ARPA repo to catch unintentionally un-awaited promises.

@trumanc you might have been around for that?